package.json

```diff
@@ -4,16 +4,16 @@
   "description": "",
   "scripts": {
     "postinstall": "tsd reinstall --overwrite --save",
-    "start": "webpack-dev-server --port 888"
+    "start": "webpack-dev-server --port 6767"
```

Comment:
For some reason port 888 is not allowed on my macbook so i changed it to something more obscure.